test/users.js

```diff
@@ -66,4 +67,46 @@ describe('/users POST', () => {
       })
       .then(() => done());
   });
+  it('should be successful signing in', (done) => {
+    userService.create({
```

Comment:
Why are you creating the users this way instead of using the dataCreation script?